packages/manager/Dockerfile

```diff
@@ -7,7 +7,8 @@ WORKDIR /app
 
 # Install dependencies based on the preferred package manager
 COPY package.json package-lock.json* ./
-RUN npm ci
+RUN npm i --force 
+#TO DO: remove this flag and chnage to npm ci
```

Comment:
Hi @nikhitamore , We've to change it in the future, Please create an issue for this. As of now we're going with it. Need to change